jujugui/static/gui/src/app/views/utils.js

```diff
@@ -570,7 +570,19 @@ YUI.add('juju-view-utils', function(Y) {
       console.log(endpoints);
       result = {error: 'Specified relation is unavailable.'};
     } else if (matches.length > 1) {
-      result = {error: 'Ambiguous relationship is not allowed.'};
+      // It's only a problem if there's more than one explicit relation.
+      // Otherwise, filter out the implicit relations and just return the
+      // explicit relation.
+      var explicitRelations = matches.filter(
+        rel => rel['interface'] !== 'juju-info');
+      if (explicitRelations.length === 0) {
+        console.log(endpoints);
+        result = {error: 'No explicitly specified relations are available.'};
+      } else if (explicitRelations.length > 1) {
+        result = {error: 'Multiple explicitly-defined, matching relations found.'};
```

Comment:
Yeah, I wanted to differentiate from the "ambiguous relation" error message above; I'll try to incorporate the familiar wording but still make it different.